go.mod

```diff
@@ -4,14 +4,16 @@ go 1.21.3
 
 require (
 	github.com/hashicorp/go-plugin v1.6.0
-	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240327133125-eed636b9a6df
+	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240515160501-a375a7ccb29e
```

Comment:
will point to master once https://github.com/smartcontractkit/chainlink-common/pull/509 is merged